### DISCUSSION TITLE
TShock Server Side Characters inventory not updating.

### DISCUSSION THREAD

**Discussion by EvanoriMemoria** [LanternPowered/Terre#13] [Q&A]
After enabling SSC in TShock the character inventory no longer updates, items cannot be removed, added, or modified, though they still can be used.

I suspect this would be fixed with the tshock-users plugin but I don't seem to be able to find the package for it. I'd greatly appreciate any insight you might have toward fixing this issue. 

Terraria 1.4.4.9
TShock 5.1.3

**Comment by Cybermaxke** [comment]
Hi, I didn't really test server side characters with the current implementation. However I have some work in progress to improve the compatibility with server side characters, which will probably resolve your issues. In the meantime I will see see if I can reproduce it and provide a quick fix.

**Reply by Cybermaxke** [reply]
The issue should be fixed now, can you give it a try?
